src/vg/civcraft/mc/citadel/CitadelConfigManager.java

```diff
@@ -83,6 +87,18 @@ public static int getMaturationScale(String type){
 		return config.getStringList("reinforcements." + type + ".lore");
 	}
 	
+	public static ReinforcementEffectType getReinforcementEffect(String type){
+		Effect effect = null; 
+		try {
+			effect = Effect.valueOf(config.getString("reinforcements." + type + ".effect.type"));
```

Comment:
You should just check whether the String you get from the config is null instead of parsing it anyway and catching the NPE. Also this will throw an IllegalArgumentException if you pass it something it can't parse (which is not null)